CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-- Update code owners (#538)
+## [0.33.0] 2022-11-22
```

Comment:
Can you add the link to the futur tag ?

Reply:
Not sure what you mean, a link to what? The github tag URL?

Reply:
Yep <https://github.com/Substra/substra-backend/releases/tag/0.33.0>